db/migrate/20200412194408_create_optional_fields.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOptionalFields < ActiveRecord::Migration[5.2]
+  def change
+    create_table :optional_fields do |t|
+      t.string :field
```

Comment:
I think we need a unique constraint on `field`. Otherwise one could have two fields with the same name

Reply:
Done